modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -194,7 +194,37 @@ With the release of the {product-short} 1.2.1 Helm chart, this is fixed.
 //link:{LinkRHIDPIssue}RHIDP-2931[RHIDP-2931]
 --
 
+== Fixed issues in {product} 1.2.2
```

Comment:
@hmanwani-rh In the RNs for 1.1, we displayed the fixed issues in reverse chronological order (as was the case in some other products that I reviewed). If we want to maintain this pracrice, this section should come immediately after the main heading see https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html/release_notes_for_red_hat_developer_hub_1.1/con-relnotes-fixed-issues_release-notes-rhdh#enhancements-and-bug-fixes-in-red-hat-developer-hub-1-1-2